src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -164,3 +164,42 @@
     margin-right: 0;
   }
 }
+
+.stem-eligibility-page {
+  max-width: 100%;
+  width: 100%;
+}
```

Comment:
Is it necessary to use this instead of `vads-u-max-width--100 vads-u-vads-u-width--full` from the [design system](https://design.va.gov/utilities/height-and-width#width)?

Reply:
changed